gradle.properties

```diff
@@ -23,7 +23,7 @@ group=org.apache.kafka
 #  - streams/quickstart/pom.xml
 #  - streams/quickstart/java/src/main/resources/archetype-resources/pom.xml
 #  - streams/quickstart/java/pom.xml
-version=3.9.1
+version=3.9.2
```

Comment:
Just curious, shouldn't this be 3.9.2-SNAPSHOT?

I'm asking this since https://github.com/apache/kafka/pull/20050 encountered below error:

```
* What went wrong:
--
more_vert | 418942 | 08:39:26 AM | Execution failed for task ':connect:json:signMavenJavaPublication'.
more_vert | 418943 | 08:39:26 AM | > Cannot perform signing task ':connect:json:signMavenJavaPublication' because it has no configured signatory
```

From build.gradle, the flag shouldSign could be skipped if the version ends with SNAPSHOT.
```
  shouldSign = !skipSigning && !version.endsWith("SNAPSHOT")
```

c.c. @showuon

Reply:
You're right! Could you help update it? Thanks.